pkg/signup/service/signup_service_test.go

```diff
@@ -669,6 +671,7 @@ func (s *TestSignupServiceSuite) TestGetSignupStatusOK() {
 			Members: []v1alpha1.Member{
 				{
 					ClusterName: "member-1",
+					ApiEndpoint: "http://api.devcluster.openshift.com",
```

Comment:
Please check that this ApiEndpoint is returned by the service and by the GET /signup endpoint.

Reply:
yep, I already did, added a slide in the review doc today
![image](https://user-images.githubusercontent.com/46766610/112206256-17a27e80-8bec-11eb-8c5c-7e9ff012f069.png)

Reply:
I mean in the unit tests :)

Reply:
Here: https://github.com/codeready-toolchain/registration-service/blob/master/pkg/signup/service/signup_service_test.go#L704
https://github.com/codeready-toolchain/registration-service/blob/master/pkg/signup/service/signup_service_test.go#L576
https://github.com/codeready-toolchain/registration-service/blob/master/pkg/signup/service/signup_service_test.go#L636
(it will test the service).
And here: https://github.com/codeready-toolchain/registration-service/blob/9efaaa9ff2bbca6f73268841eccc0944657165c4/pkg/controller/signup_test.go#L195 (will test the rest api/controller).

Reply:
i believe for the tests on L576 and L636 (bullets 2 and 3) the value of apiEndpoint should be blank, so didn't change that.
Was missing check on L704 (bullet 1) and has been added.

Reply:
> i believe for the tests on L576 and L636 (bullets 2 and 3) the value of apiEndpoint should be blank, so didn't change that.

Then verify that they are blank :)

Reply:
Yes, that was already in place.